src/processors/default/createNftFromCandyMachine.ts

```diff
@@ -29,6 +30,8 @@ export const createNftsFromCandyMachine: (metaFactory: MetaFactory) => Processor
           approved: metaFactory.autoApprove,
           burned: false,
           createdAtTime: new Date(mintTx.blockTime! * 1000),
+          chainId: metaFactory.chainId,
+          nftFactoryId: getFactoryId(metaFactory.chainId, mintAddress)
```

Comment:
will we need to backfill this nftFactoryId field on old nfts?

Reply:
yes we do this in the lens migration